package.json

```diff
@@ -370,6 +371,7 @@
     "package": "npm ci && vsce package",
     "prepare": "husky install",
     "pretest": "npm run compile && npm run lint",
+    "test": "extest get-vscode && extest install-vsix",
```

Comment:
When vsix version is not mentioned it downloads the latest version and I am fine this that for start. In the future, we may want to add another command for testing with the oldest version we consider as supported.

It is a bit strange because the `--code_version` argument seems to be supported only by get-vscode command. Not sure if that command also activates it, thus affecting the install-vsx that is run after.

I would prefer to progress on this in small/progressive steps.

Reply:
I already found how to do it but I will do it in follow-up to keep the size small. I will add another test that checks oldest version of vscode we support, it works nicely.